workspaceitems.md

```diff
@@ -151,10 +161,22 @@ An example curl call:
  -H "Content-Type:text/uri-list" \
  --data "https://dspace7.4science.it/dspace-spring-rest/api/integration/externalsources/orcid/entryValues/0000-0002-4271-0436"
 ```
-
 No confirmation, user has confirmed they want this record, and the previous state of the item is empty
 
 If an external entry value is included, the metadata from this external source should be imported automatically.  
 There's no need for a preview of the expected changes similar to the [Metadata Suggestions](metadata-suggestions.md) functionality because
-* The user has already confirmed they want to import this particular record
-* This is a new submission, it starts from an empty item
+ * The user has already confirmed they want to import this particular record
+ * This is a new submission, it starts from an empty item
+
+
+In files scenario, all works in the same way as "remote resource",  but user submits one or more file(s) instead of remote resource id.
+There are some constraints to import files:
+*   File which doesn't have a valid parser will be discarded.
+*   If multiple files are attached, only the first valid one will parsed. Others will be just saved in the WorkspaceItem.
```

Comment:
How will a user of the Angular UI know which one is "first" if they are dragging & dropping multiple files?

I wonder if we could reword this to instead be:  `If one or more files contain parseable metadata, the first file that is able to be parsed will be used to prepopulate the metadata.  Others will just be saved to the WorkspaceItem.`    That implies a small difference in behavior.  For example, if a BibTex file and Word document are drag & dropped together....it won't matter which is "first"... The Word will be unable to be parsed, so the BiBTex would be parsed instead.